src/utils.ts

```diff
@@ -500,3 +501,31 @@ export function getFileExtension(pathAsString: string) : string | undefined {
 	}
 	return undefined;
 }
+
+export function getAppendRegisteredSettingFromEnv() : string | undefined {
+	console.log(process.env);
+	const envVar = process.env[DIDACT_APPEND_REGISTERED_SETTING];
+	if (!envVar || (typeof envVar !== 'string')) {
+		return;
+	}
+	return envVar as string;
+}
+
+export async function appendAdditionalTutorialsFromEnv() : Promise<void> {
+	try {
+		const appendTutorialsAtStartup: string | undefined = getAppendRegisteredSettingFromEnv();
+		if (appendTutorialsAtStartup) {
+			await extensionFunctions.sendTextToOutputChannel(`Didact tutorials appended at startup via ${DIDACT_APPEND_REGISTERED_SETTING}`);
+			const jsonTutorials = JSON.parse(appendTutorialsAtStartup);
+			for (var i = 0; i < jsonTutorials.length; i++) {
+				const jsonObj:any = jsonTutorials[i];
+				await extensionFunctions.sendTextToOutputChannel(`--Adding tutorial ${jsonObj.sourceUri} as ${jsonObj.name}/${jsonObj.category}`);
+				await registerTutorialWithCategory(jsonObj.name, jsonObj.sourceUri, jsonObj.category);
+			}
+		}
+	} catch (ex) {
+		await extensionFunctions.sendTextToOutputChannel(ex);
+		console.error(ex);
```

Comment:
I think the `console.log` can be removed from the production code. All the more that it is written in `outputchannel`